tests/D2L.CodeStyle.TestAnalyzers.Tests/IgnoreAttribute/IgnoreAnalyzerTests.cs

```diff
@@ -37,6 +37,24 @@ class Test {
 		}
 
 		[Test]
+		public void DocumentWithoutTestIgnore_NoDiag() {
+			const string test = @"
+	using System;
+
+	namespace test {
+		[ReflectionSerializer]

```

Comment:
I think we can make this test better like so:

`csharp
public class TestAttribute : Attribute { }
public class NotTest {
	[Test]
	public void NotTestFoo() { }
}
```
... should not report diagostic